templates/openstack/index.html

```diff
@@ -109,32 +174,17 @@ <h3 class="p-matrix__title p-heading--five">Model and deploy with Juju</h3>
   {% include "shared/_resources_openstack.html"%}
 </section>
 
-<section class="p-strip is-deep is-bordered">
-  <div class="row">
-    <div class="col-8">
-      <h2>Install OpenStack now</h2>
-      <p>Use <a href="https://conjure-up.io/" class="p-link--external">conjure-up</a> to quickly deploy OpenStack on a single laptop using LXD containers, ideal for first-time users or developers. You can use the same tool with <a href="https://maas.io/" class="p-link--external">MAAS</a> to operate on racks of bare metal, or OpenStack, or VMware.</p>
-      <p><a href="/openstack/install">Install OpenStack&nbsp;&rsaquo;</a></p>
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-12">
-      <div class="u-embedded-media">
-        <iframe class="u-embedded-media__element" src="https://player.vimeo.com/video/202448226?color=ffffff&amp;title=0&amp;byline=0&amp;portrait=0" frameborder="0" allowfullscreen></iframe>
-      </div>
-    </div>
-  </div>
-</section>
-
 <section class="p-strip--light is-deep is-bordered">
```

Comment:
@steverydz either this needs to be just a p-strip, or voidbridge's logo needs to up updated to be transparent.

Reply:
@pmahnke I've made it just a p-strip - I had a look for a transparent logo but couldn't find one - even on their website